releasenotes/notes/os-capacity-94006f03f16583e4.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    This patch adds Openstack Capacity metrics and exporters to
+    StackHPC Kayobe Config. This includes a deployment playbook
+    as a hook and HAProxy configurations to support this change.
```

Comment:
I think we should note there is upgrade impact with this change, in that you need to add those new secrets to be make os-capacity work?

What happens when people start using this, and they are missing the secrets? Do we get alerts about missing scrape targets?

Reply:
Wouldn't the PrometheusTargetMissing alert pick this up if the exporter was improperly configured and crashed?

Reply:
HAProxy `HaproxyBackendDown` and `HaproxyServerDown` alerts get triggered when the auth file gets misconfigured. No Prometheus alerts get triggered and the exporter disappears form the Prometheus targets. I'll update the docs accordingly so people know what to look out for.